Python/Product/Analysis/PythonAnalyzer.cs

```diff
@@ -642,6 +642,8 @@ class LazyModuleEnumerator {
                         yield return new ExportedMemberInfo(mod.Name, name);
                     }
                 }
+
+                yield break;
```

Comment:
This change regressed at some point - if we don't break out here (in this `if` block) then we eagerly load every single module available, which adds a lot of log entries.